graphene_django/converter.py

```diff
@@ -190,19 +190,13 @@ def convert_field_to_string(field, registry=None):
     )
 
 
-@convert_django_field.register(models.BigAutoField)
 @convert_django_field.register(models.AutoField)
+@convert_django_field.register(models.BigAutoField)
+@convert_django_field.register(models.SmallAutoField)
 def convert_field_to_id(field, registry=None):
     return ID(description=get_django_field_description(field), required=not field.null)
 
 
-if hasattr(models, "SmallAutoField"):
-
-    @convert_django_field.register(models.SmallAutoField)
-    def convert_field_small_to_id(field, registry=None):
-        return convert_field_to_id(field, registry)
-
-
```

Comment:
Note that this could be considered a backwards-incompatible change as it removes the (undocumented) `convert_field_small_to_id()` function from the `graphene_django.converter` module.

So if you want, I can turn this into two PR: one only including c127aec75b33617b8eb10c84fb0733654ce70bf3 (the bugfix) and the remaining commits that fix the warnings.